src/controller/userController.js

```diff
@@ -1,11 +1,68 @@
-getUser = async (req, res, next) => {};
-addUser = async (req, res, next) => {};
-updateUser = async (req, res, next) => {};
-deleteUser = async (req, res, next) => {};
+const User = require('../models/User');
+
+const loginUser = async (req, res, next) => {
+  const { email, password } = req.body;
+  try {
+    const user = await User.find({ email, password });
+    if (user.length < 1) {
+      throw new Error('User does not exists.Please signup');
+    }
+    res.json(user);
+  } catch (err) {
+    const error = new Error(
+      `User with ${email} does not exist. Please signup.`
+    );
+    error.status = 400;
+    next(error);
+  }
+};
+
+const getUser = async (req, res, next) => {
+  const { id } = req.params;
+  try {
+    const user = await User.findById(id);
+    res.json(user);
+  } catch (err) {
+    next(err);
+  }
+};
```

Comment:
So in the end we don't need this one, right? The getUser with the id

Reply:
we can add it in case we do , right. so ill take it finally out now.